app/Models/AwardCat.php

```diff
@@ -45,11 +45,11 @@ class AwardCat extends Model
 
     protected $guarded = [];
 
-    public function awardpage(){
+    public function awardpage() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->